python/semantic_kernel/connectors/ai/google/google_ai/google_ai_prompt_execution_settings.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import sys
+from typing import Any, Literal
+
+from pydantic import Field
+
+if sys.version_info >= (3, 12):
+    from typing import override  # pragma: no cover
+else:
+    from typing_extensions import override  # pragma: no cover
+
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+
+
+class GoogleAIPromptExecutionSettings(PromptExecutionSettings):
+    """Google AI Prompt Execution Settings."""
+
+    stop_sequences: list[str] | None = Field(None, max_length=5)
+    response_mime_type: Literal["text/plain", "application/json"] | None = None
+    response_schema: Any | None = None
+    candidate_count: int | None = Field(None, ge=1)
+    max_output_tokens: int | None = Field(None, ge=1)
+    temperature: float | None = Field(None, ge=0.0, le=2.0)
+    top_p: float | None = None
+    top_k: int | None = None
+
+
+class GoogleAITextPromptExecutionSettings(GoogleAIPromptExecutionSettings):
+    """Google AI Text Prompt Execution Settings."""
+
+    pass
+
+
+class GoogleAIChatPromptExecutionSettings(GoogleAIPromptExecutionSettings):
+    """Google AI Chat Prompt Execution Settings."""
+
+    tools: list[dict[str, Any]] | None = Field(None, max_length=64)
+    tool_choice: str | None = None
```

Comment:
Tool call will be added in another PR.

Reply:
If tool calls will be added in a new PR, why include it here? Isn't it confusing for users? If you want to leave it, then should we add a model_validator that checks if they're configured and log a warning that tool calling isn't implemented?

I bring this up in the event that we need to get a package out before Google function calling is added. :)

Reply:
I will be adding function calling right after.